site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -1,27 +1,39 @@
 import { useMachine } from "@xstate/react"
```

Comment:
When the link contains an invalid parameter value, nothing is selected in the form. Should the parameter switch to the default value? Unchecked checkboxes look suspicious...

<img width="1768" alt="Screenshot 2023-07-26 at 10 49 10" src="https://github.com/coder/coder/assets/14044910/8a0e2f1e-1cd1-4e50-9185-82b0ea2e233a">

Reply:
Good catch, I agree with you.